.github/workflows/test.yml

```diff
@@ -2,19 +2,17 @@ name: CI
 
 on:
   push:
-    branches:
-    - main
   pull_request:
     branches:
     - main
 
 jobs:
   unit_tests:
     name: Unit Tests
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     strategy:
       matrix:
-        python-version: ['3.8', '3.9']
```

Comment:
Are these platform changes necessary? It's good, but it would be better to do these changes separately.

Reply:
I removed that.